src/components/kanban/KanbanBoard.vue

```diff
@@ -10,39 +10,30 @@
 -->
 <template>
   <div class="kanban-board">
-    <!-- use draggable to drag the columns -->
-    <draggable
-      v-model="columns"
-      :component-data="{ type: 'transition-group' }"
-      item-key="title"
-      class="kanban-columns"
-      :group="{ name: 'columns' }"
-    >
-      <template #item="{ element: column, index: colIndex }">
-        <KanbanColumn
-          :key="colIndex"
-          :title="column.title"
-          v-model:tasks="column.tasks"
-          :column-index="colIndex"
-          @addTask="addTask"
-          @deleteTask="deleteTask"
-          @saveTask="updateTask"
-          @update:tasks="handleTasksUpdate(colIndex, $event)"
-        />
-      </template>
-    </draggable>
+    <div class="kanban-columns">
```

Comment:
why not use draggable?

Reply:
don't need to drag columns, and they will have a stable order